float32 platform_angle
float32 manipulator_angle
